nmdc_server/ingest/omics_processing.py

```diff
@@ -106,6 +106,29 @@ def get_configuration_name(mongodb: Database, configuration_id: str, config_map)
     return config_record["name"] if config_record else None
 
 
+def get_poolable_replicate_manifest(
+    omics_processing_id: str, data_object_id: str, mongodb: Database
+) -> str:
+    """
+    Determine which poolable replicate manifest, if any, this data_generation is associated with.
+
+    Returns either the ID of a manifest of type poolable_replicates, or the ID of the
+    data_generation itself. Used for the purposes of counting data_generations for the Data Portal.
+    """
+    data_object_document = mongodb["data_object_set"].find_one({"id": data_object_id})
+    if not data_object_document or not data_object_document.get("in_manifest", None):
+        return omics_processing_id
+
+    manifest_id = data_object_document.get("in_manifest")[0]
```

Comment:
Is it possible for "in_manifest" to be empty? Could this be the hang up?

Reply:
I don't think so. I think that if `in_manifest` were empty, the condition on line 119 would be `False`, and the function would just return the data generation's id (`omics_processing_id`)